src/C5 이진탐색/P02 JS3 떡볶이 떡 만들기.js

```diff
@@ -0,0 +1,31 @@
+function binarySearch(arr, target, start, end) {
+  let result = 0;
+  while (start <= end) {
+    let total = 0;
+    let mid = parseInt((start + end) / 2);
+    arr.forEach((val) => {
+      if (val > mid) {
+        total += val - mid;
+      }
+    });
+
+    if (total < target) {
+      end = mid - 1;
+    } else {
+      result = mid;
+      start = mid + 1;
+    }
+  }
+  return result;
+}
+
+function solution(n, m, arr) {
+  arr.sort((a, b) => a - b);
+
+  return binarySearch(arr, m, 0, arr[n - 1]);
+}
+
+const N = 4,
+  M = 6;
```

Comment:
```suggestion
const N = 4;
const M = 6;
```

const 하나당 하나의 식별자만 선언하는게 더 명시적이고 직관적인거 같은데, 어떻게 생각하시나욤~~

Reply:
동의합니다. 이런 형식으로도 가능하다 라는 마음으로 사용해봤습니다 😄